Formula/r/roadrunner.rb

```diff
@@ -21,10 +21,10 @@ class Roadrunner < Formula
   def install
     ldflags = %W[
       -s -w
-      -X github.com/roadrunner-server/roadrunner/v2023/internal/meta.version=#{version}
-      -X github.com/roadrunner-server/roadrunner/v2023/internal/meta.buildTime=#{time.iso8601}
+      -X github.com/roadrunner-server/roadrunner/v2024/internal/meta.version=#{version}
+      -X github.com/roadrunner-server/roadrunner/v2024/internal/meta.buildTime=#{time.iso8601}
```

Comment:
```suggestion
      -X github.com/roadrunner-server/roadrunner/v#{version.major}/internal/meta.version=#{version}
      -X github.com/roadrunner-server/roadrunner/v#{version.major}/internal/meta.buildTime=#{time.iso8601}
```

Is there any issue with using `version.major` instead of hardcoding the year?

Reply:
good call 👍